packages/office-ui-fabric-react/src/demo/pages/DetailsListPage/examples/DetailsList.Advanced.Example.tsx

```diff
@@ -441,6 +442,11 @@ export class DetailsListAdvancedExample extends React.Component<any, IDetailsLis
   }
 
   @autobind
+  private _onItemContextMenu(item: any, index: number) {
+    console.log('Item context menu invoked', item, index);
```

Comment:
I didn't hook it up to any callbacks. I just used the same pattern that this example uses for onItemInvoked - showing how to use the property that is passed to the DetailsList. Would it be preferred if this actually showed a ContextualMenu?